paket.dependencies

```diff
@@ -7,6 +7,8 @@ storage: none
 
 group Build
 
+    framework: netstandard1.6
```

Comment:
This prevent to generate dependencies for other .NET Framework versions than the ones we projects use. You can see the chages of the paket.lock file below.